<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RFID E-Payment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>當前模式:</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="curModeLineEdit">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0,1">
            <item>
             <widget class="QPushButton" name="stdModeBtn">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>待機模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="trnModeBtn">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>交易模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="regModeBtn">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>註冊模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cnlModeBtn">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>註銷模式</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>卡片資訊</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>卡片識別 ID :</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="readRidLineEdit">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>卡片狀態 :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="readEnableLineEdit">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>卡片餘額 :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="readBalanceLineEdit">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="clearReadRidBtn">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>清除卡片識別 ID 欄位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="queryCardInfoBtn">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>重新查詢卡片資料</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTabWidget" name="funcTabWidget">
              <property name="tabPosition">
               <enum>QTabWidget::North</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="documentMode">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="trnTab">
               <attribute name="title">
                <string>交易</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
                <item>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>交易類型 :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="trnFlowComboBox">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <item>
                     <property name="text">
                      <string>儲值</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>扣款</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>交易金額 :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="trnValLineEdit">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="placeholderText">
                     <string>1-9999</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QPushButton" name="clearTrnValBtn">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>清除交易金額</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="execTrnBtn">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>執行交易程序</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="regTab">
               <attribute name="title">
                <string>註冊</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1">
                <item>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>出生年月日 (8位) :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="birthLineEdit">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="placeholderText">
                     <string>YYYYMMDD</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>身分證字號 (10位) :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="personIdLineEdit">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="placeholderText">
                     <string>A123456789</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>手機號碼 (10位) :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="phoneNumLineEdit">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="placeholderText">
                     <string>0912345678</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>卡片識別 ID :</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="generatedRidLineEdit">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="placeholderText">
                     <string>點擊下方按鈕生成識別 ID</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QPushButton" name="generateRidBtn">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>生成卡片識別 ID</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
                    <item>
                     <widget class="QPushButton" name="clearRegInfoBtn">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>清除所有欄位</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="execRegBtn">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>執行註冊程序</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="cnlTab">
               <attribute name="title">
                <string>註銷</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13" stretch="1,1">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
                    <item>
                     <widget class="QPushButton" name="setEnableFalseBtn">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>設置卡片為禁用</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="setEnableTrueBtn">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>設置卡片為啟用</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QPushButton" name="execCnlBtn">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>執行註銷程序</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="monitorTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
